doc/CMakeLists.txt

```diff
@@ -134,11 +135,15 @@ if(BUILD_DOCS AND DOXYGEN_FOUND)
   set(faqfile "${CMAKE_CURRENT_SOURCE_DIR}/faq.markdown")
   set(tutorial_path "${CMAKE_CURRENT_SOURCE_DIR}/tutorials")
   set(tutorial_py_path "${CMAKE_CURRENT_SOURCE_DIR}/py_tutorials")
+  if(BUILD_opencv_js)
+    set(CMAKE_DOXYGEN_TUTORIAL_JS_ROOT "- @ref tutorial_js_root")
+    set(tutorial_js_path "${CMAKE_CURRENT_SOURCE_DIR}/js_tutorials")
+  endif()
```

Comment:
By default, we generate OpenCV documentation for all found modules, so `BUILD_opencv_js` guard is not needed here.

Reply:
The reason is the js tutorials (except introduction tutorial) are interactive ones which include the `opencv.js` for execution in web browser. So the js tutorials depend on `opencv.js` build target. 

One solution is to put a version of `opencv.js` into js tutorials assets and remove this dependency and guard.

What do you think?

Reply:
I believe appropriate error message is enough for live demos in case of opencv.js loading error.

Reply:
OK. I will implement error message and remove the guards.